src/main/java/com/codecool/dungeoncrawl/ui/UI.java

```diff
@@ -29,6 +33,7 @@ public UI(GameLogic logic, Set<KeyHandler> keyHandlers) {
         this.context = canvas.getGraphicsContext2D();
         this.mainStage = new MainStage(canvas);
         this.keyHandlers = keyHandlers;
+        this.enemiesMoves = new SkeletonMove();
```

Comment:
This is quite an interesting idea to create a specific mover. I like it. Maybe I would introduce it on another layer. I think the mover actually does not meddle with the state of the UI rather than the "map" via the Game logic.